plugins/cargo/cargo.go

```diff
@@ -0,0 +1,22 @@
+package cargo
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func CargoCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Cargo CLI",
+		Runs:      []string{"cargo"},
+		DocsURL:   sdk.URL("https://doc.rust-lang.org/cargo/index.html"),
+		NeedsAuth: needsauth.NotForHelpOrVersion(),
```

Comment:
Not for help and for version, but I'm guessing also only for these commands:
- `cargo publish`
- `cargo owner`
- `cargo yank`